index.html

```diff
@@ -60,9 +60,162 @@ <h3>I can help you build a product , feature or website Look through some of my
         
         </section>
 
+        <section class="body2">
+            <h1 class="s2h1">My Recent Works</h1>
```

Comment:
- [ OPTIONAL] You should have only one `<h1>` tag in HTML document for semantic reasons. You have already used one in the headline section. You may use `<h2>` instead.